.gitignore

```
@@ -91,6 +91,7 @@ logs/
 mav.log
 mav.log.raw
 mav.parm
+defaults.parm
```

```suggestion
/defaults.parm
```
Should also probably do:
```
/ArduCopter/defaults.parm
/ArduPlane/defaults.parm
/AntennaTracker/defaults.parm
/ArduSub/defaults.parm
/Blimp/defaults.parm
/Rover/defaults.parm
```
For those that run out of vehicle directory.

---

Done.